docs/versioning/olm.md

```diff
@@ -0,0 +1,98 @@
+## How to create a MGC OLM bundle, catalog and how to install MGC via OLM
+
+### Creating the bundle
+    :exclamation: NOTE: You can supply different env vars to the following make commands these include:
+    * Version using the env var VERSION 
+    * Tag via the env var IMAGE_TAG for tags not following the semantic format.
+    * Image registry via the env var REGISTRY
+    * Registry org via the env var ORG
+
+    For example
+
+    `make bundle VERISON=2.0.1`
+    `make bundle IMAGE_TAG=asdf`
+    
+1. Generate the OLM bundle manifests for MGC, run the following make target:
+    ```
+    make bundle 
+    ```
+
+2. Build the bundle image
+
+    ```
+    make bundle-build
+    ```
+3. Push the bundle image to your image repository
+    ```
+    make bundle-push
```

Comment:
I wonder could we have a `make bundle-build-push`?

Reply:
sure i can create that